styleguide.setup.js

```diff
@@ -55,3 +81,4 @@ global.COMPUTATIONAL = COMPUTATIONAL
 global.PUBLICATION = PUBLICATION
 global.Checkbox = Checkbox
 global.ReactBootstrap = ReactBootstrap
+global.ReactQueryDevtoolsPanel = ReactQueryDevtoolsPanel
```

Comment:
Not used in this PR, but having this available makes it easier to debug the react-query cache for a single demo.

We don't add it to the page like the toast container because we don't want it available/visible by default.

We can't add it to the demo wrapper because one will render for each wrapper, when you only want one, and they won't be usable.

Reply:
Nice.  How would one add this ReactQueryDevtoolsPanel to debug the react-query cache for a single demo, exactly?

Reply:
You just need to add the component to the demo you're testing--it's not a perfect UX but it should be good enough for debugging

Could be worth adding a simple demo just to document that we can do this

![image](https://user-images.githubusercontent.com/17580037/130231550-77cc1fa2-3d2d-44d6-80a4-597415fb4589.png)